xbmc/ContextMenuManager.h

```diff
@@ -63,10 +68,10 @@ namespace CONTEXTMENU
   /*!
    * Starts the context menu loop for a file item.
    * */
-  bool ShowFor(const CFileItemPtr& fileItem, const CContextMenuItem& root=CContextMenuManager::MAIN);
+  bool ShowFor(const std::shared_ptr<CFileItem>& fileItem, const CContextMenuItem& root=CContextMenuManager::MAIN);
```

Comment:
Please add `#include <memory>`to this header.

Reply:
done